src/app/components/work-item-detail/work-item-detail.component.ts

```diff
@@ -367,6 +367,13 @@ export class WorkItemDetailComponent implements OnInit, OnDestroy {
           this.workItem.relationships.labels.data.map(label => {
             return this.labels.find(l => l.id === label.id);
           });
+          // Sort labels in alphabetical order
+          this.workItem.relationships.labels.data =
+          this.workItem.relationships.labels.data.sort(function(labelA, labelB) {
+            let labelAName = labelA.attributes.name.toUpperCase();
+            let labelBName = labelB.attributes.name.toUpperCase();
+            return (labelAName < labelBName) ? -1 : (labelAName > labelBName) ? 1 : 0;
```

Comment:
You could also do a `return labelAName.localCompare(labelBName)`

Reply:
Yes, I will use this. Thanks @jarifibrahim